mmv1/products/cgc/terraform.yaml

```diff
@@ -56,6 +56,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         vars:
           database_instance_name: "sqlserver-instance"
           deletion_protection: "true"
+          user: "user"
```

Comment:
We won't be able to use `random_password` for our VCR tests, so you minding these lines here for this example please?
```
        # Random provider
        skip_vcr: true
```

Reply:
Done